students/Josh_HOff/lesson06/mailroom4.py

```diff
@@ -0,0 +1,96 @@
+import pathlib
+import os
+import sys
+donors_list = {'Ralph Anders': [5, 10], 'Andrei Wasinski': [101, 151, 75], 'Stalk Holmes': [40], 'Traci Johnston': [20], 'James Hendrick': [60], 'Angelica Kisel': [45, 25, 55.60]}
+donor = ''
+donation = 0
+tab = '    '
+sorted_donors = ()
+rows = ''
+top = ''
+#this function gets a donor's name and donation amount and adds them both to the donor list.
+def send_thank_you():
+    while True:
+        print('')
+        donor = input('Type a Full Name: ')
+        truthcheck = check_input(donor)
+        if truthcheck == False:
+            return
+        elif truthcheck == True:
+            continue
+        else:
+            while True:
+                print('')
+                donation = input('What is the donation amount?: ')
+                truthcheck = check_donation(donor, donation)
+                if truthcheck == False:
+                    return
+                elif truthcheck == True:
+                    continue
+                return
+def check_input(donor):
+    if donor == 'quit':
+        return False
+    elif donor == 'list':
+        switch_func_dict.get(donor, continue_func)()
+        return True
+def check_donation(donor, donation):
```

Comment:
make sure to have 2 lines space between module functions.

Reply:
having a hard time understanding what this function does based on the name and content.

can this be refactored to be more clear? remember that your function name should reflect what it does and ideally not do too many things at once.

Reply:
> make sure to have 2 lines space between module functions.

Fixed



> having a hard time understanding what this function does based on the name and content.
> 
> can this be refactored to be more clear? remember that your function name should reflect what it does and ideally not do too many things at once.

Which function should I refactor? "check_donation"?

Reply:
Yes, I was referring to `check_donation` function.

Reply:
Got it. I've separated it into 2 functions for clarity and test-functionality.